<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Continuemission_dialog</class>
 <widget class="QWidget" name="Continuemission_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>164</y>
     <width>118</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>64</y>
     <width>284</width>
     <height>159</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/picture/setting_interface/cleardisk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>164</y>
     <width>118</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cleardisk_label">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>80</y>
     <width>228</width>
     <height>78</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>confirm_button</zorder>
  <zorder>cleardisk_label</zorder>
  <zorder>cancel_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
